tmeister/static/package.json

```diff
@@ -4,269 +4,47 @@
   "description": "",
   "main": "src/toggle-meister.js",
   "scripts": {
-    "build": "./bin/build-code",
-    "watch": "./bin/build-code -dw",
-    "postinstall": "jspm install",
-    "test": "jspmjasmine"
+    "start": "webpack-dev-server --config ./webpack.dev.js --port 8233",
```

Comment:
Does this need to be port `8233` or can we leave the explicit port out of the command as we do with our sofe service `start` commands?

Reply:
Good question. I think that might be a copy paste error I'll play with it some more.